packages/client/src/components/ContactUs/ContactUs/ContactUs.component.js

```diff
@@ -0,0 +1,177 @@
+import React, { useState, useRef } from 'react';
+import './ContactUs.style.css';
+import { Link } from 'react-router-dom';
+import getApiBaseUrl from '../../../utils/getApiBaseURL';
+import { MessageHasBeenSent } from '../MessageHasBeenSent/MessageHasBeenSent.component';
+
+export const ContactUs = () => {
+  const initialValues = { firstname: '', surname: '', email: '', message: '' };
+  const [formValues, setFormValues] = useState(initialValues);
+  const [formError, setFormError] = useState('');
+  const [status, setStatus] = useState('Submit');
+  const [isSubmit, setIsSubmit] = useState(false);
+  const [isFilled, setIsFilled] = useState(true);
+
+  const firstnameRef = useRef();
+  const emailRef = useRef();
+  const messageRef = useRef();
+
+  const setFocus = (inputValues) => {
+    if (inputValues.firstname === '') {
+      firstnameRef.current.focus();
+    } else if (inputValues.email === '') {
+      emailRef.current.focus();
+    } else if (inputValues.message === '') {
+      messageRef.current.focus();
+    }
+  };
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setFormValues({ ...formValues, [name]: value });
+  };
+
+  const handleSubmit = async (e) => {
+    if (
+      formValues.firstname === '' ||
+      formValues.email === '' ||
+      formValues.message === ''
+    ) {
+      e.preventDefault();
+      setFormError('Please fill in the required fields!');
```

Comment:
When it shows this message, the submit button goes down, which is not nice and compatible with the design. I suggest reconsideration.

Reply:
You're right. This bug happened after I clean up CSS files but didn't notice that button is moving. Will fix it in a bit:)

Reply:
Done✨